add documentation for object readyState and hideLoadImage (#60)
sassoftware/sas-viya-sdk-js: sdk/va-report-components/documentation/docs/api/ObjectHandle.md

any `ObjectHandles` obtained from that element are invalidated and should be
discarded.

## Properties

### readyState: string

> Describes the ready state of the report object. When this value changes, a `readyStateChanged` event is fired on the ObjectHandle. 

Reviewer: It seems a little odd to say "Describes" here.  Maybe just `The ready state of the report object.'?

`eventType` is a string that represents the event type to listen for. These event types are supported:
- `"selectionChanged"` for listening for selection changes in the object.

> - `"readyStateChanged"` for listening for changes in the object’s ready state.

Reviewer: ```suggestion
- `"readyStateChanged"` for listening to changes on the `readyState` property.
```

Reviewer: Seems better to directly reference the property that this change event is tracking.
